site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -445,6 +445,11 @@
 				</div>
 			#end
 		</div>
+		<div class="messageInformation"> 
```

Comment:
Should the condition go on the outside of this div? If not, then there will be a blank info box when the message text doesn't display.

Reply:
Yea good catch, I'll update that and create a new jira.

Reply:
Thanks!